### DISCUSSION TITLE
Receive automatic response on timeout even if there are no new messages

### DISCUSSION THREAD

**Discussion by elktty** [python-telegram-bot/python-telegram-bot#3369] [Q&A]
Hello!

Can I receive feedback from Telegram on timeout in any case? Whether there is a message or not. If not, then send an empty result as in the picture below.

![image](https://user-images.githubusercontent.com/89647230/202431777-53937c1f-b525-43af-bf70-dc7f5e81998b.png)

**Comment by Poolitzer** [comment]
Hey, if I guess correctly from your very cut off screenshot, you face a time out error while trying to receive new updates. The meaning of a timeout is that you loose the connection, so you can definetly not receive any information from telegram because you have no connection to it.

If you retry doing the request telegram will send all unconfirmed updates so you won't loose any information
